test_metrics.py

```diff
@@ -0,0 +1,69 @@
+import json
+import requests
+
+# This assumes a Docker container is running, with port 8030 exposed
+
+# docker build --tag dracor/metrics .
+# docker run --rm --publish 8030:8030 dracor/metrics
+
+BASE_URL = "http://localhost:8030"
+
+
+def test_root():
+    response = requests.get(BASE_URL)
+    assert response.status_code == 200
+    assert response.json() == {"service": "dracor-metrics", "version": "1.2.0"}
```

Comment:
I'd prefer to omit this test. The endpoint is purely informational, nothing is computed, and, above all, it adds the nuisance, that we would need to change the version number in two places whenever there is a version bump. I think developer convenience trumps completeness here.